Implemented dropdown for projects and work in progress on the mobile …

Hey-Zeus-byte opened the pull request:
…device NavBar
Implemented dropdown selection for "Projects" tab
Work in progress for the hamburger menu navbar
![Screen Shot 2022-09-05 at 5 18 58 PM](https://user-images.githubusercontent.com/58921019/188523520-e4b86f89-213d-482a-8e4e-4763eb5da3f7.png)
![Screen Shot 2022-09-05 at 5 19 20 PM](https://user-images.githubusercontent.com/58921019/188523524-94e2b008-e092-46e4-8f07-f15b824a3b2f.png)

fvaldez421 commented on src/components/HamburgerButton.js:
```diff
@@ -0,0 +1,56 @@
+
+const HamburgerButtonWrapper = styled.div`
+  display: block;
+  margin: 1em;
```
Why `em` here? Aren't we using `px`?

fvaldez421 commented on src/components/MobileNavLinksMenu.js:
```diff
@@ -0,0 +1,90 @@
+import {HamburgerButton} from "./HamburgerButton";
+
+const MobileNavLinkWrapper = styled.div`
+  @media only screen and (min-width: 768px) {
```
this could be changed to just `display: none;`

fvaldez421 commented on src/components/MobileNavLinksMenu.js:
```diff
@@ -0,0 +1,90 @@
+    right: 0;
+    height: 100vh;
+    width: 250px;
+    padding-top: 3.5rem;
```
Why are we using `rem` here? You also used `em`... Lets stick to one.

fvaldez421 commented on src/components/HamburgerButton.js:
```diff
@@ -0,0 +1,56 @@
+    border-radius: 5px;
+    transform-origin: 0;
+    transition: all 0.3s linear;
+    z-index: 20px;
```
`z-index` doesn't use `px`. It works in levels starting at 0 as default

fvaldez421 commented on src/components/MobileNavLinksMenu.js:
```diff
@@ -0,0 +1,90 @@
+  }
+  @media only screen and (max-width: 768px) {
+    display: flex;
+    flex-flow: column;
```
What does `flex-flow` do here? Is this somehow different than `flex-direction`?

fvaldez421 commented on src/components/MobileNavLinksMenu.js:
```diff
@@ -0,0 +1,90 @@
+    color: #000000;
+  }
+
+  @media (max-width: 768px) {
```
` only screen and `

fvaldez421 commented on src/components/MobileNavLinksMenu.js:
```diff
@@ -0,0 +1,90 @@
+  @media only screen and (max-width: 768px) {
+    display: flex;
+    flex-flow: column;
+    background-color: rgb(245, 244, 220);
```
Why are we using `rgb` here vs hex code? `#000000`

fvaldez421 commented on src/components/MobileNavLinksMenu.js:
```diff
@@ -0,0 +1,90 @@
+const DropDownContent = styled.div`
+  display: none;
+  position: absolute;
+  background-color: white;
```
lets switch this to hex code

fvaldez421 commented on src/components/NavBar.js:
```diff
@@ -28,29 +34,63 @@ const NavLink = styled(Link)`
+const DropDownContent = styled.div`
+  display: none;
+  position: absolute;
+  background-color: white;
```
lets switch this to hex code

fvaldez421 commented on src/components/HamburgerButton.js:
```diff
@@ -0,0 +1,56 @@
+  div {
+    width: 30px;
+    height: 2px;
+    background-color: black;
```
lets switch this to hex code

fvaldez421 commented on src/components/MobileNavLinksMenu.js:
```diff
@@ -0,0 +1,90 @@
+
+const NavLink = styled(Link)`
+  text-decoration: none;
+  color: #919191;
```
I would recommend we create a shared `theme` config that contains all these colors and their names/nicknames. That way we can import them across the project and be sure they're consistent.

fvaldez421 commented on src/components/HamburgerButton.js:
```diff
@@ -0,0 +1,56 @@
+import {useState} from "react";
```
didnt we talk about editing your prettier config file to use spaces between brackets? 
`{ useState }` vs `{useState}`

Hey-Zeus-byte commented:
Commit: "Added new components and updated mobile NavBar" 
- Added a DropDownButton for the sub-menu(Mobile NavBar) links for John's projects. (Work in progress: Sub menu links do not open yet).
- Mobile NavBar opens when clicking on the hamburger menu button. Error on the homepage, but i believe the Mobile Navbar doesn't sit right due to the images on homepage. 
![Screen Shot 2022-09-14 at 12 35 50 AM](https://user-images.githubusercontent.com/58921019/190090984-a182d23e-1c72-492e-8e51-abb526d171b5.png)
![Screen Shot 2022-09-14 at 12 35 58 AM](https://user-images.githubusercontent.com/58921019/190091000-26e6a91d-ac6a-44fd-8f38-74dbc536db3a.png)
